pkg/ccl/backupccl/backup_test.go

```diff
@@ -922,9 +923,16 @@ func backupAndRestore(
 			rows, idx, bytes int64
 		}
 
-		backupURIFmtString, backupURIArgs := uriFmtStringAndArgs(backupURIs)
+		backupURIFmtString, backupURIArgs := uriFmtStringAndArgs(backupURIs, 0)
 		backupQuery := fmt.Sprintf("BACKUP DATABASE data INTO %s", backupURIFmtString)
-		sqlDB.QueryRow(t, backupQuery, backupURIArgs...).Scan(
+		kmsURIArgs := make([]interface{}, 0)
+		var kmsURIFmtString string
+		if kmsURIs != nil {
```

Comment:
nit: it's generally safer to check the length of a slice in go instead of emptiness. Suppose the caller of this function passed in zero length kmsURIs, then we'd try to create the kms uri below leading to sadness. This is inconsequential here, but thought i'd share on my reviewing pedestal.

Reply:
oh, neat! for some reason "slices are always pointers" went to "empty slices are always nil" in my mind, but of course that's ridiculous. Thanks!